sdk/ai/azure-ai-client/azure/ai/client/aio/operations/_patch.py

```diff
@@ -6,9 +6,216 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List
+import logging
+from typing import List, AsyncIterable
+from ._operations import EndpointsOperations as EndpointsOperationsGenerated
+from ...models._patch import EndpointProperties
+from ...models._enums import AuthenticationType, EndpointType
+from ...models._models import ConnectionsListSecretsResponse, ConnectionsListResponse
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+logger = logging.getLogger(__name__)
+
+
+class InferenceOperations:
+
+    def __init__(self, outer_instance):
+        self.outer_instance = outer_instance
+
+    async def get_chat_completions_client(self) -> "ChatCompletionsClient":
```

Comment:
get embedding client method is named as "get_async_embeddings_client". This one is "get_chat_completions_client", is there a reason being not consistent?

Reply:
Oh, forgot to change the name back to the original "get_embeddings_client". Thanks @jhakulin

Reply:
Should this "get_async_azure_openai_client" be also then without async?